src/main/java/cceclipseplugin/core/CCIgnore.java

```diff
@@ -125,6 +128,10 @@ public void loadCCIgnore(IProject p) {
 	 * 
 	 */
 	private void serverCleanUp(IProject p) {
```

Comment:
no, but this is getting opened in a different issue b/c it's out of scope for this